service/client_state.hh

```diff
@@ -68,6 +68,9 @@ public:
         UNINITIALIZED, AUTHENTICATION, READY
     };
 
+    /// Conversion to string according to C* convention in `system.clients`.
+    sstring to_cassandra_string(auth_state as);
```

Comment:
This should either be static, or use client_state::_auth_state

Reply:
Or, even better, removed. (anyway `auth_state` cannot be handled properly at this point, because it changes after connection)